File: solutions/beecrowd/1039/1039.py
import math
import sys

for line in sys.stdin:
    r1, x1, y1, r2, x2, y2 = map(float, line.split())
    distance = math.sqrt((x1 - x2) ** 2 + (y1 - y2) ** 2)

    if r1 >= distance + r2:
        print('RICO')
    else:
        print('MORTO')

File: solutions/beecrowd/1039/1039.clj
(ns main
  (:require [clojure.string :as str]))

(defn main []
  (loop [line (read-line)]
    (when line
      (let [[r1 x1 y1 r2 x2 y2] (->> (str/split line #" ")
                                     (map #(Integer/parseInt %)))
            distance (Math/sqrt
                      (+
                       (Math/pow (- x1 x2) 2)
                       (Math/pow (- y1 y2) 2)))]
        (if (>= r1 (+ distance r2))
          (println "RICO")
          (println "MORTO")))
      (recur (read-line)))))

(main)
